hmda-quarterly-data-service/src/test/scala/hmda/quarterly/data/api/route/denials/DenialRatesByTypeLoanPurposeHomeSpec.scala

```diff
@@ -0,0 +1,20 @@
+package hmda.quarterly.data.api.route
+
+import org.scalatest.{Matchers, WordSpec}
+import hmda.quarterly.data.api.route.rates.denials.DenialRatesByTypeLoanPurposeHome
+
+
+class DenialRatesByTypeLoanPurposeHomeSpec extends WordSpec with Matchers {
+  val route = DenialRatesByTypeLoanPurposeHome.getRoute
+  val routeSummary = DenialRatesByTypeLoanPurposeHome.getSummary
+  "denial rates by type loan purpose home route" should {
+    "return the correct summary route" in {
+      assert(!routeSummary.isCompleted)
+    }
+  }
+  "denial rates by type loan purpose home route" should {
+    "have a string title" in {
```

Comment:
For what looks like all of the tests in this PR, you can move this into the "should" section above it and delete this "should" section, since the titles are the same. So something like:
```
"denial rates by type loan purpose home route" should {
    "return the correct summary route" in {
      assert(!routeSummary.isCompleted)
    }

    "have a string title" in {
       assert(route.title.isInstanceOf[String])
    }
  }
}
```

Reply:
I agree, but I think I want to change this in the next sprint since it isn't breaking anything / will require a lot of moving around. So I'll make note of this and fix it with my data publisher tests.